src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandHandlerTests.cs

```diff
@@ -46,8 +46,8 @@ public class CreateInvitationCommandHandlerTests
         private const string _mcPkgNo2 = "MC2";
         private const string _commPkgNo = "Comm1";
         private const string _commPkgNo2 = "Comm2";
-        private const string _system = "1|2";
```

Comment:
keep _system and _system2 and rename them to systemPathWithoutSection and create a positive test